src/combinatorics/simple_graph/basic.lean

```diff
@@ -1,11 +1,13 @@
 /-
 Copyright (c) 2020 Aaron Anderson, Jalex Stark, Kyle Miller. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Aaron Anderson, Jalex Stark, Kyle Miller, Alena Gusakov
+Authors: Aaron Anderson, Jalex Stark, Kyle Miller, Alena Gusakov, Hunter Monroe
 -/
+
```

Comment:
The [style guide](https://leanprover-community.github.io/contribute/style.html) specifies no line break here.  (Same for the subgraph module.)

Reply:
WIll fix